web_sdk/web_test_utils/lib/image_compare.dart

```diff
@@ -137,3 +163,49 @@ Golden file $filename did not match the image generated by the test.
   }
   return 'OK';
 }
+
+Future<void> _uploadToSkiaGold(
+  SkiaGoldClient skiaClient,
+  Image screenshot,
+  String filename,
+) async {
+  // Can't upload to Gold Skia unless running in LUCI.
+  assert(_isLuci);
+
+  // Write the screenshot to the file system so it can be consumed by the
+  // `goldctl` tool.
+  final file.File goldenFile = fs.file(filename);
```

Comment:
If I'm reading the code correctly the `filename` is the name provided by the test. Does it mean that this file will be written to CWD? I think we should output to `.dart_tool`.

Reply:
Nice catch! I'll put it under the `skia_gold` folder.